mobile/lib/modules/sharing/views/album_viewer_page.dart

```diff
@@ -184,6 +183,7 @@ class AlbumViewerPage extends HookConsumerWidget {
 
     Widget _buildImageGrid(AlbumResponseDto albumInfo) {
       if (albumInfo.assets.isNotEmpty) {
+        print(albumInfo.assets[2].exifInfo);
```

Comment:
Remove print statement here

Reply:
Sorry, been using a lot of print statements for debugging. This was so I could figure out why there was no EXIF data for assets in an album.